packages/core/src/module-resolver.ts

```diff
@@ -159,7 +159,11 @@ export type SyncResolverFunction<R extends ModuleRequest = ModuleRequest, Res ex
 ) => Res;
 
 export class Resolver {
-  constructor(readonly options: Options) {}
+  componentCache: Map<String, any>;
```

Comment:
Why `any`? 

Wouldn't that work? 👇

```suggestion
  componentCache: Map<String, { hbsModule?: string, jsModule?: string }>;
```